dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/AccessLogData.java

```diff
@@ -67,17 +67,17 @@ private void setPair(int index, long l) {
     }
 
     private String getStringValue(int index) {
```

Comment:
should move getStringValue / getIntValue / getLongValue into GenericData.  We can then remove those methods from all of the GenericData subclasses